src/kernel/content/format/cms_formats.e

```diff
@@ -26,6 +25,8 @@ feature -- Access
 
 	all_formats: LIST [CONTENT_FORMAT]
 		once
+				-- Can we provide an external file to read the
```

Comment:
maybe not external file, but module and maybe via database

side note: for such comment, prefix with FIXME, TODO , NOTE or QUESTION